pkg/crc/oc/clusteroperator.go

```diff
@@ -25,19 +25,27 @@ type K8sResource struct {
 	} `json:"items"`
 }
 
-func GetClusterOperatorStatus(oc OcConfig) (bool, error) {
-	allAvailable := true
+type ClusterStatus struct {
```

Comment:
Do you expect to have multiple fields `true` at the same time? Or could it be an enumeration?
```
type ClusterStatus int
const (  
        Available = ClusterStatus iota
        Degraded
        Progressing
)
```

Reply:
@cfergeau we can have multiple fields `true` at the same time, there is a chance when an operator is available but also degrading like it need more replica but only few are available.